diff --git a/node_modules/simple-json-db/index.d.ts b/node_modules/simple-json-db/index.d.ts
index 2c945f1..e30ec76 100644
--- a/node_modules/simple-json-db/index.d.ts
+++ b/node_modules/simple-json-db/index.d.ts
@@ -1,13 +1,13 @@
-declare class JSONdb {
+declare class JSONdb<T = object> {
   constructor(filePath: string, options?: { asyncWrite?: boolean, syncOnWrite?: boolean });
 
-  set(key: string, value: object) : void;
-  get(key: string) : object | undefined;
+  set(key: string, value: T) : void;
+  get(key: string) : T | undefined;
   has(key: string) : boolean;
   delete(key: string) : boolean | undefined;
   deleteAll() : this;
   sync() : void;
-  JSON(storage?: object) : object;
+  JSON(storage?: Record<string, T>) : Record<string, T>;
 }
 
 export = JSONdb;
